x/monitoringp/types/genesis_test.go

```diff
@@ -49,6 +49,8 @@ func TestGenesisState_Validate(t *testing.T) {
 					1000,
 					"foo", // chain id should be <chain-name>-<revision-number>
 					sample.ConsensusState(0),
+					1,
+					spntypes.DefaultUnbondingPeriod,
```

Comment:
Order seems to be reversed here:

```suggestion
					spntypes.DefaultUnbondingPeriod,
					1,
```